src/views/blocks/tickets/submit-button.php

```diff
@@ -23,14 +26,30 @@
  * @param string $button_name The button name. Set to cart-button to send to cart on submit, or set to checkout-button to send to checkout on submit.
  */
 $button_name = apply_filters( 'tribe_tickets_ticket_block_submit', 'cart-button' );
+
+/**
+ * Allow filtering of the button classes for the tickets block.
+ *
+ * @since TBD
+ *
+ * @param array $button_name The button classes.
+ */
+$button_classes = apply_filters(
+	'tribe_tickets_ticket_block_submit_classes',
+	[
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 3e25065ee413a465ebce8d72fdc7c357</sub>